### DISCUSSION TITLE
Example Armadillo dropping with "sharp" edge

### DISCUSSION THREAD

**Discussion by blower05** [InteractiveComputerGraphics/PositionBasedDynamics#141] [Q&A]
Hi Prof Bender,

I tried to modify the rigid body to fall on into a relatively 'sharp edge' actually it is not sharp at all and actually It is just a plank. The armadillo soft body can poke through the plank during the simulation. I tried subdivided the plank for more meshes but it is still the same. I tried 105 substeps and collision tolerance with different very small and large values, it still failed the collision. What should I do to correct this?
BTW, I tried a cube of wider width and the collision is OK and looks normal.

Thanks
![sharp_edg_failed2](https://user-images.githubusercontent.com/113827342/192507514-a1a2de0e-dfcd-4365-96db-f82438bef09c.PNG)
![sharp_edge_failed1](https://user-images.githubusercontent.com/113827342/192507524-01e3cff6-6dce-4000-82ed-7e8f2c053f1f.PNG)

**Comment by janbender** [comment]
The simulator checks all the box SDF for all aramdillo vertices and vice versa if the setting is correct in the scene file. Can you post the scene?
